config/src/config/partial.rs

```diff
@@ -76,6 +80,17 @@ pub struct Storage {
     pub db_path: Option<PathBuf>,
 }
 
+/// Protocol-specific configuration
+#[derive(Deserialize, Default, Debug, Clone, PartialEq)]
+pub struct Protocol {
+    /// :D
+    #[serde(default)]
+    pub epoch_zero_timestamp: Option<i64>,
+    /// ;D
+    #[serde(default)]
+    pub checkpoint_period: Option<u64>,
```

Comment:
I think this should have a `#[serde(rename = "checkpoint_period_seconds")]` like the rest of time parameters

Reply:
:+1: I renamed the field to `checkpoint_period_seconds`.